docs/source/training/training.rst

```diff
@@ -0,0 +1,787 @@
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+      Besides the machine description URIs you provided in the previous exercise, NICE
+      requires some extra input to run. We focus on the inverse mode of NICE for now.
+      For this mode NICE requires the following:
+
+      - A desired plasma shape
+      - The plasma current
+      - Characteristics of the vacuum toroidal field; R0 and B0
+      - p' and ff' profiles
+
+      First, open the ``Plasma Shape`` options menu, set it to ``parameterized``, and
+      leave the shape settings on theirs defaults for now.
+
+      Secondly, open the ``Plasma Properties`` options menu, and set it to the ``Manual`` option. 
+      Leave the values at its default for now. This will set the plasma current, R0 and B0, and the ff' and p' profiles
+      through :ref:`a parameterisation <abg_parameterisation>` using the alpha, beta, and gamma parameters. Leave the values
+      at default for now.
+
+      You should now have set up enough to run NICE inverse mode, which you can verify by
+      checking that there are no more ⚠️ icons besides the option menus, and that the ``Run`` button is enabled.
+
+      What do you see in the plot on the left? What happens if you hover your mouse over the 
+      coil outlines? Try playing with the settings in the ``Plotting Parameters`` menu. What do they do?
+
+
+   .. md-tab-item:: Solution
+      
+      If NICE inverse converged with your desired plasma shape, you will see the resulting 
+      equilibrium contour lines appear on the plot on the left. 
+
+      When you hover over the coil outlines, you will see the currents calculated by NICE. 
+
+      Using the ``Plotting Parameters``
+      options, you can change how many contour lines are plotted, as well as change which 
+      plotting components are shown.
+
+      For example: 
+
+      .. image:: ../images/training/nice_result.png
+         :align: center
+
+Exercise 6c: Configurating the Plasma Shape
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+
+      There are three ways to configure the desired plasma shape for NICE inverse in the Plasma Shape Editor.
+
+      1. By providing an equilibrium IDS containing a `boundary outline <https://imas-data-dictionary.readthedocs.io/en/latest/generated/ids/equilibrium.html#equilibrium-time_slice-boundary-outline>`_.
+      2. By providing a geometric parameterization.
+      3. By providing gap distances for an equilibrium IDS containing `boundary gaps <https://imas-data-dictionary.readthedocs.io/en/latest/generated/ids/equilibrium.html#equilibrium-time_slice-boundary-gap>`_.
+      
+      We will try out the first two methods in this exercise.
+
+      1. Select the ``Equilibrium IDS outline`` option. 
+         Try to provide an outline from an equilibrium IDS, for example by using the URI below
+         if you are on SDCC. Try to visualize the boundary outline of the time steps at 200s and 251s, 
```

Comment:
> Do or do not, there is no try.

Please review all places where you use `try`, it sounds a bit childish in a training (unless you want people to try something to figure out it's not possible for some reason)